src/k8s/pkg/k8sd/database/clusterconfigs/cluster_configs.go

```diff
@@ -51,6 +52,11 @@ type APIServer struct {
 	DatastoreClientKey  string `yaml:"datastore-client-key,omitempty"`
 }
 
+type K8sDqlite struct {
+	Host string `yaml:"host,omitempty"`
```

Comment:
This is not ClusterConfiguration, as it will be different on each node. Instead, change this to specify the subnet (e.g. `193.168.q.0/24`)

We either set this specify this during bootstrap or autodetect based on the default node address.

Also, make shre to use IP addresses for k8s-dqlite, as we have seen hostnames cause subtle issues in the past, especially around the interface used by dqlite.